iiab-support

```diff
@@ -46,14 +49,17 @@ fi
 if grep -q '^openvpn_install: True' /etc/iiab/local_vars.yml; then
     echo -e "Your IIAB installation appears normal, with OpenVPN already installed...\n"
 else
-    echo -e "Please wait a few minutes as IIAB Stage 1 (1-prep) & OpenVPN are installed...\n"
+    echo -e "Plz wait a few minutes as sshd, iiab-admin & OpenVPN are confirmed/installed...\n"
     if grep -q '^openvpn_install:' /etc/iiab/local_vars.yml; then
         sed -i "s/^openvpn_install:.*/openvpn_install: True/" /etc/iiab/local_vars.yml
     else
         echo "openvpn_install: True" >> /etc/iiab/local_vars.yml
     fi
     cd /opt/iiab/iiab
-    ./runrole 1-prep
+    export ANSIBLE_LOG_PATH="/opt/iiab/iiab/iiab-install.log"
```

Comment:
if you use the full path the cd is not required

Reply:
Good to know.  I've commented it out.

Reply:
Here